tests/epyccel/test_arrays.py

```diff
@@ -13,7 +13,7 @@
 def test_array_int32_1d_scalar_add(language):
 
     f1 = arrays.array_int32_1d_scalar_add
-    f2 = epyccel( f1 , language = language)
+    f2 = epyccel( f1 , language = language, verbose=True)
```

Comment:
Is the `verbose` flag useful in your opinion, or is it just a remnant of your debugging efforts?

Reply:
Your merge caused a RuntimeError. I put the verbose in to try to find it but it doesn't seem to be reproducible :(

Reply:
Removed in afaafad